controllers/netbot.js

```diff
@@ -4272,6 +4276,50 @@ class netBot extends ControllerBot {
 
   }
 
+  /*
+  value:[
+    strict single api message obj  {
+      "mtype": "cmd",
+      "data": {
+        "value": {
+          "featureName": "adblock"
+        },
+        "item": "enableFeature"
+      },
+      "type": "jsonmsg",
+      "target": "0.0.0.0"
+		}
+  ]
+
+  */
+  batchHandler(gid, rawmsg, callback) {
+    (async () => {
+      const batchActionObjArr = rawmsg.message.obj.data.value;
+      const copyRawmsg = JSON.parse(JSON.stringify(rawmsg));
+      const results = [];
+      for (const obj of batchActionObjArr) {
+        obj.type = "jsonmsg"
+        obj.data.ignoreRate = true;
+        copyRawmsg.message.obj = obj;
+        let result, error;
+        try {
+          result = await this.msgHandlerAsync(gid, copyRawmsg);
```

Comment:
`msgHandler` calls `cmdHandler` essentially, and that latter calls `this.simpleTxData` on every command, this should not happen.

Reply:
call `msgHandlerAsync` will pass a callback to simpleTxData, and it will trigger callback and return directly. I tested it and works fine.

Reply:
`msgHandler` will call `this.simpleTxData ` without a callback and send data to the cloud, but `msgHandlerAsync ` will pass a callback, just resolve promise and return a result for every single action, like create policy or something else.